DemoqaProject/config/StoreElementTexts.cs

```diff
@@ -8,5 +8,24 @@ public class StoreElementTexts
         public static string widgetsText = "Widgets";
         public static string interactionText = "Interactions";
         public static string bookText = "Book Store";
+
+        public static string fullName = "Andrew Garfield";
+        public static string email = "Andrew@gmail.com";
+        public static string currentAddress = "Wall Street 23";
+        public static string permanentAddress = "Wisconsin Dells 18";
+        
+        public static string invalidEmail="Andrew123";
+        
+        public static string fNamePH = "Full Name";
+        public static string emailPH = "name@example.com";
+        public static string currentPH = "Current Address";
+        
+        public static string firstName = "Denzel";
+        public static string lastName = "Washington";
+        public static string emailForm = "Denzel@gmail.com";
+        public static string age = "67";
+        public static string salary = "3500";
+        public static string department = "New York, Mount Vernon";
```

Comment:
You shouldn't store static data for all tests in one place.
If you use some const values for tests -- store them in POM class as const values.
You can see that some string duplicates each other and you need to invent a new name for it.
If there really some common parameters/data then you can store it in base page, in other way -- please don't do it.

Reply:
done